.github/workflows/cd-pr.yml

```diff
@@ -0,0 +1,37 @@
+name: Deploy PR-Preview
```

Comment:
Let's rename cd-pr.yml as deploy-pr-preview.yml and ci as build-pr-preview.yml

CI/CD name is confusing as other workflow is also a part of CI

Reply:
changed the name